src/screens/Onboarding/WelcomeScreen.tsx

```diff
@@ -49,8 +49,10 @@ const WelcomeScreen = ({
 
   return (
     <View style={styles.container}>
-      <FullScreenHeader setFocusOnLoad={false} color={themeColor} />
-      <Illustration Svg={Onboarding1_2} />
+      <View style={styles.illustration}>
```

Comment:
Kunne man lagt dette inn i Illustration-komponenten, for mer gjenbruk?

Reply:
ja, vet ikke hvor mye gjenbruk vi får ut av det, siden komponenten blir mer spesifikk, men vi er jo forsåvidt der allerede at den er spesifikk for onboarding.

Reply:
etter å ha tittet litt mer på det tenker jeg at komponenten blir mindre gjenbrukbar vet at vi enforcer disse stilene på illustration.
Jeg merger den derfor slik den er.